README.md

```diff
@@ -32,4 +32,10 @@ var app = new EmberApp({
 });
 ```
 
+## Troubleshooting
+
+You must have phantomjs installed in your system for the tests to run.
```

Comment:
I don't think this is true anymore, I can run them fine without it.

Reply:
It is if you don't have chrome installed, which I didn't since I use chromium under Linux.